html/semantics/forms/textfieldselection/initial-selection.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Initial cursor position in text entry controls</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#textFieldSelection">
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div id="container">
+  <input type="text" value="value" autofocus>
+  <input type="text" value="value">
+  <input type="search" value="value">
+  <input type="tel" value="value">
+  <input type="password" value="value">
+  <textarea>hello</textarea>
+</div>
+
+<script>
+"use strict";
+test(() => {
+  for (const el of Array.from(document.querySelector("#container").children)) {
```

Comment:
Is the Array.from really needed?  I'd hope everyone does iterable nodelists by now....  I guess it doesn't really hurt to much.